modules/virt-hot-plugging-a-virtual-disk.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
+
+[id="virt-hot-plugging-a-virtual-disk_{context}"]
+= Hot-plugging a virtual disk
```

Comment:
Consider changing the title to "Hot-plugging a virtual disk using the CLI"

Reply:
Changed titles and file names of both topics.